providers/apis/defi/raydium/types.go

```diff
@@ -80,7 +80,7 @@ var DefaultAPIConfig = oracleconfig.APIConfig{
 	Enabled:          true,
 	Name:             Name,
 	Timeout:          500 * time.Millisecond,
-	Interval:         100 * time.Millisecond,
+	Interval:         500 * time.Millisecond,
```

Comment:
why this change?

Reply:
Based on our current RPC providers, we'll get rate-limited otherwise